mashed-potatoes.md

```diff
@@ -0,0 +1,66 @@
+Mashed Potatoes
+======
+
+## Ingredients
+Makes 4 servings.
+- 2lb Yukon Gold Potatoes
+- 8 tbsp Unsalted Butter (1 stick)
+- 1/2 cup whole milk
+- Salt + Pepper to taste
+
+## Tools
+- Potato masher
+- Potato peeler
+- Pot big enough to hold potato mixture
+- Colander
+- Spatula
+
+## Steps
+1. Peel potatoes. Put peeled potatoes into cold water bath to prevent oxidation.
+1. Cut potatoes into even sized cubes (for faster cooking). Put into cold water bath.
+1. Rinse starch off cubed potatoes.
+1. Get butter out of fridge and cut into smaller pieces (~1 tbsp pieces works). Leave out to get butter to room temp.
+1. In pot, put potatoes and enough cold water to fully submerge the potatoes. Have a little excess water.
+1. Add a good pinch of salt to potato + water mixture.
+1. Turn heat to HIGH to boil the pot. Cover to reach boiling faster.
+    * Careful that pot does not overflow during boil
+1. Once boiling, lower heat to LOW and make sure potatoes are simmering.
+1. Simmer until potatoes break apart easily with fork. Usually ~20 minutes.
+1. Drain potatoes using colander. Leave draining for ~5 minutes to release excess moisture.
+1. Clean out pot of all residue.
+1. Put potatoes back in pot and put pot on stove on LOW-MED heat.
+1. Use masher to break up the cubed potatoes into smaller chunks.
+1. Add in salt and ground black pepper.
+1. Drop in 2-4 pieces of butter and mash.
+1. Repeat until all butter pieces are incorporated. If you want more of a whipped texture, use spatula to incorporate instead of mashing. Overmashing will cause the potato to stick together more and create a consistency you may not want (honestly I'm sure the difference is that much...).
```

Comment:
```suggestion
1. Repeat until all butter pieces are incorporated. If you want more of a whipped texture, use spatula to incorporate instead of mashing. Overmashing will cause the potato to stick together more and create a consistency you may not want (honestly I'm not sure the difference is that much...).
```